### ISSUE TITLE
[Feature Request] Allow Click on Task to also Highlight where it is and position it

### ISSUE THREAD

**Issue by Obadiahcruz** [delashum/obsidian-checklist-plugin#58]
Hello, I absolutely love the work you have done with this. I would like to suggest the ability to show a highlight of where the task is inside the note. As it is now, it just opens the note. it would be nice if it could implement what the backlinks do, which is also highlights the part where it is inside the note and it also moves the note position to where it is.

**Comment by delashum** [comment]
Yes agreed, this would be awesome. I'll look into how this could be accomplished

**Comment by Obadiahcruz** [comment]
Thank You! Game changer for sure! Great Work

**Comment by rui-nar** [comment]
I would REALLY love to have this as well. I have usually very long notes and it's quite complicated to find the task item in there.
Cheers
